prediction_market_agent_tooling/markets/omen/data_models.py

```
@@ -1,8 +1,8 @@
 import typing as t
 from datetime import datetime
-
+from typing_extensions import Self
 import pytz
-from pydantic import BaseModel
+from pydantic import BaseModel, Field, ConfigDict, model_validator, computed_field
```

Ruff

`pydantic.model_validator` imported but unused

Remove unused import

(F401)